Modules/_datetimemodule.c

```diff
@@ -1836,7 +1813,7 @@ microseconds_to_delta_ex(PyObject *pyus, PyTypeObject *type)
     PyObject *num = NULL;
     PyObject *result = NULL;
 
-    tuple = checked_divmod(pyus, us_per_second);
+    tuple = PyNumber_Divmod(pyus, us_per_second);
```

Comment:
I haven't looked closely at any of the other stuff, but +1 on the `_datetimemodule.c` stuff. It is very nice that it is now possible to make this simplification.